src/domains/transaction/pages/SendVoteTransaction/SendVoteTransaction.test.tsx

```diff
@@ -100,31 +101,197 @@ describe("Vote For Delegate", () => {
 		);
 		const { getByTestId, asFragment } = render(
 			<FormContext {...form.current}>
-				<FourthStep delegate={delegate} transaction={transaction!} />
+				<FourthStep transaction={transaction!} unvotes={[]} votes={votes} />
 			</FormContext>,
 		);
 
 		expect(getByTestId("TransactionSuccessful")).toBeTruthy();
 		expect(asFragment()).toMatchSnapshot();
 	});
 
+	it("should send a unvote & vote transaction", async () => {
+		const history = createMemoryHistory();
+		const voteTransactionURL = `/profiles/${fixtureProfileId}/wallets/${wallet.id()}/transactions/vote/`;
+
+		const params = new URLSearchParams({
+			unvotes: delegateData[1].address,
+			votes: delegateData[0].address,
+		});
+
+		history.push({
```

Comment:
Why would this information be attached to the URL instead of being passed to a component through properties? The URL has no use for this information and there is no use for that information outside of a specific component.

Reply:
For communication between uncorrelated pages in the tree, and also "ready" for aip-26: https://github.com/ArkEcosystem/AIPs/blob/master/AIPS/aip-26.md#vote

Reply:
The `SendVoteTransaction` page is accessed by `Votes` and `WalletVote`. But in the new designer it's accessed only by `Votes`, so in another PR I can remove this route and use it as a component of `Votes` and pass the data of `votes/unvotes` through properties. What do you think?